include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,553 @@
+    //VkBool32           workgroupMemoryExplicitLayout;
+    //VkBool32           workgroupMemoryExplicitLayoutScalarBlockLayout;
+    //VkBool32           workgroupMemoryExplicitLayout8BitAccess;
+    //VkBool32           workgroupMemoryExplicitLayout16BitAccess;
+    
+    /* VK_KHR_zero_initialize_workgroup_memory *//* MOVED TO Vulkan 1.3 Core */
+    /* VK_KHX_multiview *//* see VK_KHR_multiview *//* MOVED TO Vulkan 1.1 Core */
+
+    /* ComputeShaderDerivativesFeaturesNV *//* VK_NV_compute_shader_derivatives */
+    //VkBool32           computeDerivativeGroupQuads;
+    //VkBool32           computeDerivativeGroupLinear;
+
+    /* CooperativeMatrixFeaturesNV *//* VK_NV_cooperative_matrix */
+    //VkBool32           cooperativeMatrix;
+    //VkBool32           cooperativeMatrixRobustBufferAccess;
+
+    /* CornerSampledImageFeaturesNV *//* VK_NV_corner_sampled_image */
+    //VkBool32           cornerSampledImage;
+
+    /* CoverageReductionModeFeaturesNV *//* VK_NV_coverage_reduction_mode */
+    //VkCoverageReductionModeNV                        coverageReductionMode;
+
+    /* DedicatedAllocationImageAliasingFeaturesNV *//* VK_NV_dedicated_allocation_image_aliasing */
+    //VkBool32           dedicatedAllocationImageAliasing;
+
+    /* DiagnosticsConfigFeaturesNV *//* VK_NV_device_diagnostics_config */
+    //VkBool32           diagnosticsConfig;
+
+    /* DeviceGeneratedCommandsFeaturesNV *//* VK_NV_device_generated_commands */
+    //VkBool32           deviceGeneratedCommands;
+
+    /* ExternalMemoryRDMAFeaturesNV *//* VK_NV_external_memory_rdma */
+    //VkBool32           externalMemoryRDMA;
+    
+    /* FragmentShaderBarycentricFeaturesNV *//* VK_NV_fragment_shader_barycentric */
+    //VkBool32           fragmentShaderBarycentric;
+
+    /* FragmentShadingRateEnumsFeaturesNV *//* VK_NV_fragment_shading_rate_enums */
+    //VkBool32           fragmentShadingRateEnums;
+    //VkBool32           supersampleFragmentShadingRates;
+    //VkBool32           noInvocationFragmentShadingRates;
+
+    /* InheritedViewportScissorFeaturesNV *//* VK_NV_inherited_viewport_scissor */
+    //VkBool32           inheritedViewportScissor2D;
+
+    /* LinearColorAttachmentFeaturesNV *//* VK_NV_linear_color_attachment */
+    //VkBool32           linearColorAttachment;
+    
+    /* MeshShaderFeaturesNV *//* VK_NV_mesh_shader */
+    //VkBool32           taskShader;
+    //VkBool32           meshShader;
+    
+    /* RayTracingMotionBlurFeaturesNV *//* VK_NV_ray_tracing_motion_blur */
+    //VkBool32           rayTracingMotionBlur;
+    //VkBool32           rayTracingMotionBlurPipelineTraceRaysIndirect;
+
+    /* RepresentativeFragmentTestFeaturesNV *//* VK_NV_representative_fragment_test */
+    //VkBool32           representativeFragmentTest;
+    
+    /* ExclusiveScissorFeaturesNV *//* VK_NV_scissor_exclusive */
+    //VkBool32           exclusiveScissor;
```

Comment:
add as a TODO to expose (don't expose right now, as it would require extra API work to use)

in GL/GLES this is `NV_scissor_exclusive`